airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/steps/python_registry/utils.py

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+
+from urllib.parse import urlparse
+
+import requests
+from pipelines.airbyte_ci.steps.python_registry.context import PythonPackageMetadata
+
+
+def is_package_published(package_metadata: PythonPackageMetadata, registry_url: str) -> bool:
```

Comment:
To boost re-use of this function could be split the args and not use `PythonPackageMetadata` here?
The signature could be `name, version, registry_url`. 
You can store this module under `airbyte-ci/connectors/pipelines/pipelines/helpers/pip.py`

Wdyt?